### PR TITLE
10.0 tests payment term type alxolivares

### PR DESCRIPTION
**Pull request by alxolivares**
@nhomar tests para addons-vauxoo/payment-term-type.

### CODE REVIEW

File: payment_term_type/tests/test_payment_term_type.py [hunk 1]
```diff
@@ -10,6 +10,13 @@ class TestPaymentTermType(TransactionCase):
         payment terms line to compute
     """
 
+    def setUp(self):
+        """Seudo-constructor method"""
```
Comment (nhomar) [inline payment_term_type/tests/test_payment_term_type.py]: remove useless documentation

File: payment_term_type/tests/test_payment_term_type.py [hunk 2]
```diff
@@ -27,3 +34,76 @@ def test_payment_term_type_credit(self):
...
+        }
+        return self.config_obj.create(values)
+
+    def _create_payment_term(self):
+        """Creates a test payment term."""
+        values = {
+            'company_id': self.company.id,
+            'name': 'Test payment term',
+        }
+        return self.pay_term_obj.create(values)
```
Comment (nhomar) [inline payment_term_type/tests/test_payment_term_type.py]: no extra methods for creation

File: payment_term_type/tests/test_payment_term_type.py [hunk 3]
```diff
@@ -27,3 +34,76 @@ def test_payment_term_type_credit(self):
...
+        )
+
+    def test_03_payment_term_type_bdp_cash(self):
+        """This test validates that payment type is equal to cash if payment
+        terms is based on date of payments and days equal to 0
+        """
+        self._create_config_setting('bdp')
+        pay_term_id = self._create_payment_term()
+        self.assertEqual(
+            pay_term_id.payment_type, 'cash', 'Unexpected value.'
```
Comment (nhomar) [inline payment_term_type/tests/test_payment_term_type.py]: better messaging

File: payment_term_type/tests/test_payment_term_type.py [hunk 4]
```diff
@@ -27,3 +34,76 @@ def test_payment_term_type_credit(self):
...
+        pay_term_id.write({
+            "line_ids": [
+                (0, 0, {
+                    'value': 'percent',
+                    'value_amount': 0.0,
+                    'days': 15,
+                    'sequence': 1,
+                })
+            ]
+        })
```
Comment (nhomar) [inline payment_term_type/tests/test_payment_term_type.py:122]: tests without asserts are not tests

**Review by nhomar — CHANGES_REQUESTED** [review]

**Comment by alxolivares** [comment]
@nhomar all requested changes have been made.

**Comment by nhomar** [comment]
Can you make a proper PR to the customer repository pliz before merge.

**Comment by alxolivares** [comment]
@nhomar This is the dummy for the customer repository vauxoo/imeqmo#49

**Review by nhomar — APPROVED** [review]

**Pull request merged**